components/participant_table/commons/participant_table_base.lua

```diff
@@ -299,14 +305,17 @@ function ParticipantTable:getPlacements()
 	Array.forEach(mw.ext.LiquipediaDB.lpdb('placement', {
 		limit = 5000,
 		conditions = '[[placement::!]] AND [[pagename::' .. string.gsub(mw.title.getCurrentTitle().text, ' ', '_') .. ']]',
-	}), function(placement) placements[placement.opponentname] = true end)
+	}), function(placement) placements[placement.opponentname] = placement end)
 
 	return placements
 end
 
 ---@param lpdbData table
 ---@return string
 function ParticipantTable:objectName(lpdbData)
+	local objectName = Logic.emptyOr(lpdbData.objectName, lpdbData.objectname)
```

Comment:
Why is this needed?

Reply:
because the first is when reading from vars (sc/sc2/sg) while the 2nd is when reading from lpdb